dist/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <script src="index.js" type="module" defer></script>
+  <!-- <script src="https://cdn.tailwindcss.com"></script> -->
```

Comment:
- [ ] To keep your code clean, kindly get rid of any commented code or not helpful comments in your repo 🙏 
- [ ] you guys have an error in your website caused by calling index.js in your <head> tag in the index.html file , besides that the file src is wrong, since we are using webpack, it will be responsible for calling the index.js or (main.js as you guys decided to name it) inside the HTML file, so just remove the importing line `<script src="index.js" type="module" defer></script>` and the error will be fixed 👌 

![image](https://user-images.githubusercontent.com/89282221/197281420-3b2cb85e-8dae-47f0-a757-0586a7f7fd88.png)